source/core_mqtt.c

```diff
@@ -1264,6 +1286,7 @@ static MQTTStatus_t sendPublishAcks( MQTTContext_t * pContext,
     uint8_t packetTypeByte = 0U;
     MQTTPubAckType_t packetType;
     MQTTFixedBuffer_t localBuffer;
+    MQTTConnectionStatus_t connectStatus;
```

Comment:
Why is this a parameter? Seems its just written to.

Reply:
I am not sure what you mean exactly but it is read on line number 1313 and 1315.